api/endpoints.go

```diff
@@ -74,7 +74,7 @@ func (a *api) handleGETHostState(c jape.Context) {
 		return
 	}
 
-	a.writeResponse(c, http.StatusOK, HostState{
+	a.writeResponse(c, http.StatusOK, HostState(HostState{
```

Comment:
Wrapping the type with `HostState()` is unnecessary since it's already a `HostState{}`.